language/tools/vm-genesis/src/lib.rs

```diff
@@ -215,19 +215,6 @@ fn create_and_initialize_main_accounts(
             )
         });
 
-    // create the core code account
```

Comment:
what was this used for before?

Reply:
We created an account and rotated its pubkey to be a random value so that it's not spendable by anyone IIRC.